Solutions/NetClean ProActive/Analytic Rules/NetClean_Sentinel_analytic_rule.yaml

```diff
@@ -0,0 +1,42 @@
+id: 77548170-5c60-42e5-bdac-b0360d0779bb
+name: NetClean ProActive Incidents
+description: |
+  'NetClean Incident'
+severity: High
+requiredDataConnectors: []
+status: Available 
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: GreaterThan
+triggerThreshold: 0
+suppressionDuration: 5h
+tactics:
+  - Discovery
+relevantTechniques:
+  - T1083
+query: |
+  Netclean_Incidents_CL |  where version_s == 1
+entityMappings:
+  - entityType: FileHash
+    fieldMappings:
+      - identifier: Value
+        columnName: md5_s
+      - identifier: Algorithm
+        columnName: MD5
+  - entityType: DNS
+    fieldMappings:
+      - identifier: DomainName
+        columnName: domain_s
+  - entityType: Host
+    fieldMappings:
+      - identifier: HostName
+        columnName: Hostname_s
```

Comment:
Is this of the form username@domain.com?
If yes, split this into 2 parts and the username is the hostname while domain.com is the domain.